src/main/java/org/imsglobal/caliper/Sensor.java

```diff
@@ -97,7 +97,8 @@ public CaliperClient getClient(String key) {
      * @return envelope
      */
     public Envelope create(String id, DateTime sendTime, String dataVersion, List<CaliperSendable> data) {
-        return new Envelope(id, DateTime.now(), dataVersion, data);
+        DateTime time = sendTime == null ? DateTime.now() : sendTime;
```

Comment:
this is standard ims global code which we shouldnt be changing, if you feel the need to change it to add this logic, i would suggest it be handled in the code that builds the caliper message.

Reply:
This does look like an omission, as the provided argument (sendTime) is ignored in favour of generating a new time.
I've rolled back the Sensor change and put a workaround in the test.

Please review if the above is an omission, and refactor accordingly.